tests/integrations/resource_metering/test_dynamic_config.rs

```diff
@@ -150,7 +150,7 @@ pub fn case_precision(test_suite: &mut TestSuite) {
     test_suite.start_agent_at(port);
```

Comment:
Should we also rename all "agent" in the code into "receiver", in order to avoid confusing? Since they are the same thing.

/cc @zhongzc @crazycs520

Reply:
@LemonHX s/agent/receiver/g

Reply:
https://github.com/tikv/tikv/pull/10975#discussion_r714606238

Reply:
> Should we also rename all "agent" in the code into "receiver", in order to avoid confusing? Since they are one thing.
> 
> /cc @zhongzc @crazycs520

globally?!